n3fit/src/evolven3fit/eko_utils.py

```diff
@@ -0,0 +1,59 @@
+from ekobox import gen_theory, gen_op
+from eko import run_dglap
+
+from validphys.loader import Loader
+from . import utils
+
+
+def construct_eko_cards(theoryID, op_card_dict, t_card_dict, q_fin, q_points, x_grid):
+    """Return the theory and operator cards used to construct the eko"""
+    # theory_card construction
+    theory = Loader().check_theoryID(theoryID).get_description()
```

Comment:
We are going to have a "chicken and egg" problem here since we need to generate the eko when we generate the theory... (right now the theories always came first so it's ok, we will see in the future).

Reply:
Well but we don't need need to construct the full theory to produce the eko, right? We just need to make sure that `Loader().check_theoryID(theoryID).get_description()` provides us the theory information (so I believe it is enough to have an "empty" theory with only the info).

Reply:
If possible, I'd tackle directly the issue at the database level: you want the information there, no more, and that should be accessible without the full theory (if it does not exist there, we can just say that theory does not exist).

Problem is that I'd like to be able to consume that database in a simple way. An API would be wonderful, but even too much for what we need (it's a small file after all).
Can we just transform the SQLite database to a downloadable JSON file? (if possible automatically update)
That would be simple enough:
1. download the json from a URL (or get it from the repo, if we have it)
2. load the json
3. take the value corresponding to the ID of your theory

Reply:
It would be even reasonable to drop SQLite at all and go for JSON directly, but in this PR I'd limit the impact on everything else.

Reply:
> I'd tackle directly the issue at the database level:

Agreed.

> It would be even reasonable to drop SQLite at all and go for JSON directly, but in this PR I'd limit the impact on everything else.

Yes. This is not for this PR or for any in the short future. But it is something to keep in mind.